packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -979,7 +978,7 @@ export default class Meeting extends StatelessWebexPlugin {
    */
   verifyPassword(password, captchaCode) {
     return this.fetchMeetingInfo({
-      destination: this.sipUri, type: _SIP_URI_, password, captchaCode
+      destination: this.sipUri, type: this.type, password, captchaCode
```

Comment:
Thanks for making this changes so going forward for type conversations also the verifyPassword will work